backend/src/main/java/it/chalmers/gamma/controller/WhitelistController.java

```diff
@@ -29,9 +31,9 @@
     private final WhitelistService whitelistService;
     private final ActivationCodeService activationCodeService;
     private final MailSenderService mailSenderService;
-
+    private static final Logger LOGGER = LoggerFactory.getLogger(WhitelistController.class);
     @Value("${mail.receiver.standard-postfix}")
-    private static final String MAIL_POSTFIX = "@student.chalmers.se";
+    private static final String MAIL_POSTFIX = "student.chalmers.se";
```

Comment:
Are you sure this doesn't break anything anywhere else?

Reply:
ja

Reply:
it's added at another place, should not be in the string at all